app.js

```diff
@@ -2,22 +2,120 @@
   if (!window.addEventListener) return // Check for IE9+
 
   let options = INSTALL_OPTIONS
+  let updateTimeout
+  const delay = 1500
```

Comment:
Use `UPPER_SNAKE_CASE` for constants that are "sources of truth" to the app. Additionally, change the name to describe what the delay is for, e.g. `UPDATE_DELAY`